getReviews.py

```diff
@@ -1,5 +1,6 @@
 import csv
 import io
+from os import name
```

Comment:
You have a variable conflict with "name" being overwritten for different things (ex. line 220).  Also, not seeing where os.name is being used.  Would either delete, or if still needed, should just alter to `import os` and use as `x = os.name`.